packages/launch/src/focused-launch/index.tsx

```diff
@@ -112,7 +129,7 @@ const FocusedLaunch: React.FunctionComponent< Props > = ( { siteId } ) => {
 				<div className="focused-launch__data-input">
 					<div className="focused-launch__section">
 						<label className="focused-launch__label">
-							{ __( '3. Confirm your plan', __i18n_text_domain__ ) }
+							{ stepNumber++ }. { __( 'Confirm your plan', __i18n_text_domain__ ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.